packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -1,10 +1,11 @@
 const WORDPRESS_NAMESPACE = '@wordpress/';
 const BUNDLED_PACKAGES = [
+	'@wordpress/dataviews',
 	'@wordpress/icons',
+	'@wordpress/interactivity',
```

Comment:
I sorted this list and added `@wordpress/interactivity`. Because it's compiled as a module, it isn't suitable for use as an external script and should be bundled in scripts, not externalized.

Reply:
`@wordpress/interactivity` gets bundled in the view files with these changes if I follow it correctly:
<img width="814" alt="Screenshot 2024-01-02 at 09 46 47" src="https://github.com/WordPress/gutenberg/assets/699132/b4bfedc6-cf51-4212-bab2-1f81a1a36a3c">

Reply:
I'll remove the change to add this. It's unrelated.

Reply:
Done.